src/openrct2/paint/paint.h

```diff
@@ -123,6 +123,15 @@ typedef struct sprite_bb {
     rct_xyz16 bb_size;
 } sprite_bb;
 
+typedef struct paint_struct_bound_box {
```

Comment:
Maybe its better to move the struct into paint.h and also replace the members of paint_struct with it, later we can make a simple copy from it.